### PR TITLE
Move flag to painless tests

### PR DESCRIPTION
**Pull request by nik9000**
The `-XX:-OmitStackTraceInFastThrow` flag is only required by Painless's
tests so we'll only set it there. This is much simpler.

### CODE REVIEW

File: buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy [hunk 1]
```diff
@@ -468,18 +468,7 @@ class BuildPlugin implements Plugin<Project> {
             File heapdumpDir = new File(project.buildDir, 'heapdump')
             heapdumpDir.mkdirs()
             jvmArg '-XX:HeapDumpPath=' + heapdumpDir
-            /*
-             * We only want to append -XX:-OmitStackTraceInFastThrow if a flag for OmitStackTraceInFastThrow is not already included in
-             * tests.jvm.argline.
-             */
             final String testsJvmArgline = System.getProperty('tests.jvm.argline')
```
Comment (jasontedor) [inline buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy]: We still need to set the `argLine` property.

**Review by jasontedor — CHANGES_REQUESTED** [review]
We still need to set the `argLine` property or the system property doesn't percolate down.

File: buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy (hunk 1 above)
Comment (nik9000) [inline buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy]: sorry! coming.

**Comment by nik9000** [comment]
@jasontedor I pushed a fix.

**Review by jasontedor — APPROVED** [review]
LGTM.

**Comment by jasontedor** [comment]
Thank you @nik9000. ❤️

**Pull request merged**